### PR TITLE
clean blacklist

### PR DESCRIPTION
**Pull request by rvalieris**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

I going to try to clean the blacklist a bit here. xref: https://github.com/bioconda/bioconda-recipes/pull/4972

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks for working on this @rvalieris!!!

File: sorted-r-blacklist [hunk 1]
```diff
@@ -58,20 +58,16 @@ recipes/bioconductor-bubbletree
 # TODO: bioconductor-cexor needs specific version of genomicranges?
 #recipes/bioconductor-cexor
 
-
+# conflicts with recent Rcpp version
+recipes/r-histonehmm
```
Comment (bgruening) [inline sorted-r-blacklist]: can we pin an older version, or do we need to have an older version available for R3.3.2 before :(

**Comment by bgruening** [comment]
This is fantastic! Feel free to merge, I will drink a juice and celebrate you now!

File: sorted-r-blacklist (hunk 1 above)
Comment (rvalieris) [inline sorted-r-blacklist]: the main problem is that I have no idea which version of Rcpp it needs, the dev site is not explicit either..

**Comment by rvalieris** [comment]
not so fast ! there are a few more recipes I want to give it a go

**Comment by bgruening** [comment]
You could do this in a second PR isn't it? Tests are taking longer and longer :)
Your call! :)

**Comment by rvalieris** [comment]
oh ! I didnt noticed, just this one more then I will merge.

**Comment by bgruening** [comment]
Thanks @rvalieris!

**Pull request merged**